test/unit/components/compose-comment.test.jsx

```diff
@@ -152,26 +163,20 @@ describe('Compose Comment test', () => {
                 }
             }
         });
-        const component = mountWithIntl(
-            <ComposeComment
-                {...defaultProps()}
-                isReply
-            />
-            , {context: {store}}
-        );
-        expect(component.find('FlexRow.compose-comment').exists()).toEqual(false);
-        expect(component.find('MuteModal').exists()).toBe(true);
-        expect(component.find('MuteModal').props().startStep).toBe(1);
-        expect(component.find('CommentingStatus').exists()).toEqual(false);
+        const {container, findByComponentName} = getComposeCommentWrapper({isReply: true}, store);
+        expect(container.querySelector('.compose-comment')).not.toBeInTheDocument();
+        expect(screen.getByRole('dialog')).toBeInTheDocument();
```

Comment:
This test seems semantically different from the old test. Should this be a `querySelector` for `.modal-mute` like above, or maybe a `findByComponentName` as in the next line, or is this better than the old test in some way?